CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# dbt_dynamics_365_crm v0.1.0
```

Comment:
We'll want this first release to be a pre-release. Can you update this to be `v0.1.0-a1`